sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/TimeSeriesInsightsLifecycleSamples.cs

```diff
@@ -28,7 +28,7 @@ public async Task RunSamplesAsync()
                 #region Snippet:TimeSeriesInsightsGetModelSettings
 
                 // Get the model settings for the time series insights environment
-                Response<TimeSeriesModelSettings> currentSettings = await client.GetModelSettingsAsync();
+                Response<TimeSeriesModelSettings> currentSettings = await client.ModelSettings.GetModelSettingsAsync();
```

Comment:
Wondering if API names need update now since it is kind of evident. ModelSettings.GetAsync() will also suffice.

Reply:
I agree, it seems redundant at this point.